Description: No relative path for require in tests
Let the interpreter find the library with loadpath in tests
So that the tests can be run with the installed package (autpkgtest)
Author: Cedric Boutillier <boutil@debian.org>
Forwarded: no
Last-Update: 2022-01-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/test_pygments.rb
+++ b/test/test_pygments.rb
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 require 'test/unit'
-require File.join(File.dirname(__FILE__), '..', '/lib/pygments.rb')
+require 'pygments'
 ENV['mentos-test'] = 'yes'
 
 P = Pygments
